src/tests/ftest/util/server_utils_params.py

```diff
@@ -501,6 +501,7 @@ def __init__(self, base_namespace, index, provider=None, max_storage_tiers=MAX_S
             if name in self.REQUIRED_ENV_VARS:
                 default_env_vars.extend(self.REQUIRED_ENV_VARS[name])
         self.env_vars = BasicParameter(None, default_env_vars)
+        self.env_pass_through = BasicParameter(None, None)
```

Comment:
If we update the systemctl env with LD_LIBRARY_PATH, then we need to pass through LD_LIBRARY_PATH here or otherwise it won't work as expected